student.rb

```diff
@@ -0,0 +1,14 @@
+require './person'
```

Comment:
- [ ] OPTIONAL
By convention, it's a good practice to use `require_relative` classes in the same source and use `require` for external modules or packages.

Reply:
Thanks a lot I'll do that!